### PR TITLE
Remove clarketm from CODEOWNERS

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,9 @@
 *                                    @istio/wg-test-and-release-maintainers
-.prow.yaml                           @clarketm @cjwagner @fejta @michelle192837 @chases2 @istio/wg-test-and-release-maintainers
-/authentikos/                        @clarketm @cjwagner
-/prow/                               @clarketm @cjwagner @fejta @michelle192837 @chases2
+.prow.yaml                           @cjwagner @fejta @michelle192837 @chases2 @istio/wg-test-and-release-maintainers
+/authentikos/                        @cjwagner
```
Comment (brian-avery) [inline CODEOWNERS]: This line worries me. Should we add a second person?

Comment (ericvn) [reply CODEOWNERS]: T&R maintainers?

Comment (brian-avery) [reply CODEOWNERS]: Done. I wasn't sure if there was a reason that it was only specific people.

**Comment by brian-avery** [comment]
Spoke to @clarketm. He said that it was ok to remove him from the prow files, but asked to leave him on authentikos.

**Review by clarketm — APPROVED** [review]

**Comment by howardjohn** [comment]
@clarketm aren't you in the T&R as well anyways? or did you leave that?

**Comment by brian-avery** [comment]
He did give approval on behalf of T&R. Lol.

**Comment by clarketm** [comment]
> @clarketm aren't you in the T&R as well anyways? or did you leave that?

I am still a member via peribolos job. I will send out a PR to [`istio/community`](https://github.com/istio/community/blob/8a69309d4546467100c8f14a5f11d8d1d5f77954/org/istio.yaml#L964) to adjust my membership.

**Pull request merged**